static_src/test/unit/stores/user_store.spec.js

```diff
@@ -682,6 +682,19 @@ describe('UserStore', function () {
     });
   });
 
+  describe('getAllInOrgAndNotSpace()', function() {
+    it('should find all users that are in an org, without the space users', function() {
```

Comment:
I am probably misreading this spec, but from the description, shouldn't the `testUser` be excluded here? They don't have a `space_roles` key on their object, they do have a space role in their `roles`.

Should the corresponding method in `UserStore` also check for the existence of space roles within the `roles` object?